data/templates/Detector_ArcK8s.csx

```diff
@@ -0,0 +1,26 @@
+private static string GetQuery(OperationContext<AzureCosmosDB> cxt)
+{
+    return
+    $@"
+        let startTime = datetime({cxt.StartTime});
+        let endTime = datetime({cxt.EndTime});
+        cluster('CdbSupport').database('Support').<YOUR_TABLE_NAME>
+        | where Timestamp >= startTime and Timestamp <= endTime
+        | <YOUR_QUERY>";
+}
+
+[ArmResourceFilter(provider: "Microsoft.Kubernetes", resourceTypeName: "connectedClusters")]
+[Definition(Id = "<YOUR_DETECTOR_ID>", Name = "", Author = "<YOUR_ALIAS>", Description = "")]
+public async static Task<Response> Run(DataProviders dp, OperationContext<AzureCosmosDB> cxt, Response res)
```

Comment:
AzureCosmosDB? Should this is be ARMResource

Reply:
Fixed